src/main/java/org/gradle/profiler/GradleBuildInvocationResult.java

```diff
@@ -12,9 +12,48 @@
 public class GradleBuildInvocationResult extends BuildInvocationResult {
     private final Duration garbageCollectionTime;
     private final Duration timeToTaskExecution;
+    private final Duration gradleToolingAgentExecutionTime;
+    private final Duration studioExecutionTime;
     private final Map<String, Duration> cumulativeBuildOperationTimes;
     private final String daemonPid;
 
+    public GradleBuildInvocationResult(
+        BuildContext buildContext,
+        Duration executionTime,
+        Duration gradleToolingAgentExecutionTime,
+        Duration ideExecutionTime,
+        @Nullable Duration garbageCollectionTime,
+        @Nullable Duration timeToTaskExecution,
+        Map<String, Duration> cumulativeBuildOperationTimes,
+        String daemonPid
+    ) {
+        super(buildContext, executionTime);
+        this.garbageCollectionTime = garbageCollectionTime;
+        this.timeToTaskExecution = timeToTaskExecution;
+        this.gradleToolingAgentExecutionTime = gradleToolingAgentExecutionTime;
+        this.studioExecutionTime = ideExecutionTime;
```

Comment:
```suggestion
        this.ideExecutionTime = ideExecutionTime;
```

This field should probably be renamed as well.

Reply:
renamed

Reply:
Done